patoolib/programs/p7rzip.py

```diff
@@ -18,8 +18,28 @@
 7zr is a light executable supporting only the 7z archive format.
 """
 
-from .p7zip import \
-  extract_7z, \
-  list_7z, \
-  test_7z, \
-  create_7z
+from .p7zip import create_7z
+
+def extract_7z(archive, compression, cmd, verbosity, interactive, outdir):
```

Comment:
neither commit nor PR describes the need of duplicating those functions here, instead of reusing from p7zip.  Please elaborate

Reply:
To elaborate on the commit message, 7zr does not support the '-p-' argument which is used in the p7zip functions.